File: test/eu/shooktea/passkeeper/PasswordGeneratorTest.scala
package eu.shooktea.passkeeper

import org.junit.{Test, Assert}

class PasswordGeneratorTest {

  @Test
  def generateRandomPassword() : Unit = {
    val passwordLength: Int = 15
    val password: String = PasswordGenerator.generate(passwordLength)
    Assert.assertEquals(passwordLength, password.length)
  }
}

File: test/eu/shooktea/passkeeper/PasswordGeneratorTest.java
package eu.shooktea.passkeeper;

import static org.junit.Assert.*;
import org.junit.Test;

public class PasswordGeneratorTest {

    @Test
    public void generateRandomPassword() {
        int passwordLength = 15;
        String password = PasswordGenerator.generate(passwordLength);
        assertEquals(passwordLength, password.length());
    }
}
